projects/vanilla/src/app/configurators/facet-configurator.component.ts

```diff
@@ -82,8 +95,12 @@ export class FacetConfiguratorComponent implements OnChanges {
     }
     // Update the list of available aggregations
     this.aggregations = this.searchService.results?.aggregations
-      .filter(agg => !!agg.isTree === (this.config.type === 'facet-tree'))
       .map(agg => agg.name) || [];
+
+    if (this.config.parameters?.aggregation) {
+      const column = this.appService.getColumn(this.config.parameters.aggregation);
```

Comment:
Attention car `appService.getColumn()` attend le nom d'un champ, et là tu lui passes le nom d'une aggrégation (qui ne sont pas forcément les mêmes).
Une façon plus simple à mon avis:
`this.isTree = !!this.aggregations.find(a => a.name === this.config.parameters?.aggregation)?.isTree`